sdk/communication/Azure.Communication.Sms/src/Models/SmsSendResponse.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.Sms.Models
+{
+    [CodeGenModel("SmsSendResponse")]
+    internal partial class SmsSendResponse
```

Comment:
will this make it hidden even though it's in the models folder?

Reply:
Yea, because it is internal, it will be accessible only to other code in the SMS project